conda-forge.yml

```
@@ -1,5 +1,5 @@
 conda_build:
-  error_overlinking: true
+  error_overlinking: false
```

Are we able to revert this after the `patchelf` change?

```suggestion
  error_overlinking: true
```

---

No, we've needed this anytime we update rpath (example: https://github.com/conda-forge/libcurand-feedstock/pull/2/files#diff-478786365dd93f740eb520c2faa03d7a0623273663b75472272c8ef94297bbe2)

This will be the error:
```
WARNING (cuda-sanitizer-api,compute-sanitizer/libsanitizer-public.so): runpaths ['$ORIGIN/../lib'] found in /home/conda/feedstock_root/build_artifacts/cuda-sanitizer-api_1706216449799/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/compute-sanitizer/libsanitizer-public.so

```